Makefile.am

```
@@ -65,6 +65,8 @@ shared_libshared_la_SOURCES = \
 	shared/missing.h \
 	shared/strbuf.c \
 	shared/strbuf.h \
+	shared/tmpfile-util.c \
+	shared/tmpfile-util.h \
```

we only care about meson right now.

---

If we don't add these two files to Makefile.am, Autotools will not successfully compile, and CI will not pass. Are we going to ignore this error?

---

ok. Just make it part of the commit adding the file rather than as the last commit. If this goes in before #289 , then I rebase it that will drop the entire thing.